pkg/3scale/amp/component/system.go

```diff
@@ -192,7 +191,7 @@ func (system *System) buildSystemBaseEnv() []v1.EnvVar {
 		helper.EnvVarFromSecret("USER_EMAIL", SystemSecretSystemSeedSecretName, SystemSecretSystemSeedAdminEmailFieldName),
 		helper.EnvVarFromSecret("TENANT_NAME", SystemSecretSystemSeedSecretName, SystemSecretSystemSeedTenantNameFieldName),
 
-		helper.EnvVarFromValue("THINKING_SPHINX_ADDRESS", "system-sphinx"),
+		helper.EnvVarFromValue("THINKING_SPHINX_ADDRESS", SystemSphinxServiceName),
```

Comment:
Abstracting this out is great, do you think it makes sense to push the name change of  `system-searchd` now a lot of changes are done or leave it for later?

Reply:
we can, but it will require more work. For instance, the service called `system-sphinx` would need also to be updated (removed and recreated as CR names cannot be changed). Same as the env var `THINKING_SPHINX_ADDRESS`, we could change that to `THINKING_SEARCHD_ADDRESS`, but that would imply update the configmap and Porta's code as well. 

I would not change all this in this PR. But, at the same time, to be honest, if we leave that for later, it will probably never be done. It is a good exercise for somebody getting familiar with the operator code, but there are other task with higher priorities.

Reply:
I will give it a try